ui/app/lib/kv-breadcrumbs.js

```diff
@@ -13,13 +13,13 @@ function pathIsFromDirectory(path) {
   return path ? !!path.match(/\//) : false;
 }
 
-function breadcrumbsForDirectory(path) {
+function breadcrumbsForDirectory(path, noLinkToLastItem) {
```

Comment:
```suggestion
function breadcrumbsForDirectory(path, noLinkToLastItem = false) {
```

Reply:
It might be easier to understand that by default if `noLinkToLastItem` is not passed in, it defaults to false.